packages/opentelemetry-api/src/trace/span.ts

```diff
@@ -47,14 +47,17 @@ export interface Span {
    * Sets a single Attribute with the key and value passed as arguments.
    *
    * @param key the key for this attribute.
-   * @param value the value for this attribute.
+   * @param value the value for this attribute. Setting a value null or
+   *              undefined is invalid and will result in undefined behavior.
```

Comment:
Nit: could you indent this second line of the JSDoc by 4 spaces to make to be consistent with others, also to make visually clear that it's a continued line.